modules/tooltip/js/tooltip_directive.js

```diff
@@ -170,6 +171,7 @@
                 });
 
                 setTooltipPosition();
+                angular.element(document).on('scroll', setTooltipPosition);
```

Comment:
Shouldn't we debounce it a bit? We have a `LxUtils.debounce` we can use.

Reply:
Right !